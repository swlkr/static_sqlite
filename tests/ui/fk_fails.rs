use static_sqlite::sql;

sql! {
    let migrate = r#"
         create table User (
             id integer primary key,
             email text unique not null
         );

         create table Todo (
             id integer primary key,
             user_id integer not null references Users(id)
         );
     "#;
}
