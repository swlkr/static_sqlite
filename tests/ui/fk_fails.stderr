error: MissingTable("Users")
 --> tests/ui/fk_fails.rs:4:9
  |
4 |     let migrate = r#"
  |         ^^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
  --> tests/ui/fk_fails.rs:15:2
   |
15 | }
   |  ^ consider adding a `main` function to `$DIR/tests/ui/fk_fails.rs`
